### PR TITLE
Minor and Cosmetic Improvements

### PR DESCRIPTION
**Pull request by TheOneric**
- All text files should end with newlines
- More portable `run-docker-build.sh`
- Clarify and correct some thing in the Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+and enables you to get consistent results in authoring and web-playback, provided libass is also used locally.
 
 [ONLINE DEMO](https://libass.github.io/JavascriptSubtitlesOctopus/videojs.html) / [other examples with demo](https://libass.github.io/JavascriptSubtitlesOctopus/)
 
 ## Features
 
-- Supports all SSA/ASS features
-- Supports any fonts (including woff2 fonts)
+- Supports most SSA/ASS features (everything libass supports)
+- Supports all OpenType- and TruType-fonts (including woff2 fonts)
```
Comment (TFSThiagoBR98) [inline README.md:12]: @TheOneric TrueType?

Comment (TheOneric) [reply README.md]: thanks, fixed.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
 [![Actions Status](https://github.com/libass/JavascriptSubtitlesOctopus/workflows/Emscripten/badge.svg)](https://github.com/libass/JavascriptSubtitlesOctopus/actions)
 
-SubtitlesOctopus displays subtitles in .ass format and easily integrates with HTML5 videos. It supports all SSA/ASS features and fully compatible with [libass](https://github.com/libass/libass).
+SubtitlesOctopus displays subtitles in .ass format and easily integrates with HTML5 videos.
+Since it utilises [libass](https://github.com/libass/libass), SubtitlesOctopus supports most SSA/ASS features
```
Comment (astiob) [inline README.md:4]: Very minor and arguable nit: “utilises” is a slightly unusual and flavoured word; I’d just write “uses”, which is more common and neutral.

I couldn’t put my finger on it myself, but my dictionary says:
> make practical and effective use of:
> > <i>vitamin C helps your body utilize the iron present in your diet</i>

**Review by astiob — COMMENTED** [review]
Readme-only review:

Looks good.

Maybe also add libass to the <i>Included Libraries</i> list for good measure?

File: README.md (hunk 2 above)
Comment (TheOneric) [reply README.md]: Changed to `uses` and added libass to included libraries, thanks!

**Review by TFSThiagoBR98 — APPROVED** [review]

**Pull request merged**